### PR TITLE
A15-13629 Add Gift Wrapped Test

### PR DESCRIPTION
**Pull request by danggrianto**
@sophiahoeke not sure if the data need to be validated as well. see the TODO

### CODE REVIEW

File: fixtures/items.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    "merchandiseClass": "8192",
+    "skuId": "string",
+    "lastModified": 0,
+    "firstAdded": 0,
+    "categoryId": "category-123",
+    "productId": "product-123",
+    "giftWrap": {
+        "skuId": "139093",
+        "code": "FP_GIFT_WRAP_A",
```
Comment (OSeraph) [inline fixtures/items.py]: I think you'll need to account for the gift wrap options from the other brands as well

**Comment by danggrianto** [comment]
the source code on the pricing service doesn't differ those. But I spoke with @sophiahoeke she will do the variations.

**Comment by OSeraph** [comment]
@danggrianto okay in that case +1

File: fixtures/items.py (hunk 1 above)
Comment (danggrianto) [reply fixtures/items.py]: like i said, this doesn't really matter, the scope of the function is only moving this request body. but i will need to confirm with paul first about the scope of the tests, some of the function is still hardcoded.

**Pull request merged**